newtest/reloader_test.go

```diff
@@ -0,0 +1,255 @@
+package newtest
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/mesosphere/dkp-test-framework/pkg/addons"
+	"github.com/mesosphere/dkp-test-framework/pkg/cluster"
+	"github.com/mesosphere/dkp-test-framework/pkg/cluster/kind"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	typedcorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+)
+
+func deployKubeaddonsController(cluster cluster.Cluster) error {
+	return addons.HelmInstall(cluster, "kubeaddons", "https://mesosphere.github.io/kubeaddons/chart", "kubeaddons", "kube-system", nil)
+}
+
+func deployReloader(restConfig rest.Config) error {
+	restConfig.ContentConfig.GroupVersion = &schema.GroupVersion{Group: "kubeaddons.mesosphere.io", Version: "v1beta1"}
+	client, err := rest.RESTClientFor(&restConfig)
```

Comment:
Was https://github.com/mesosphere/dkp-test-framework/blob/main/pkg/addons/kubeaddons.go#L21 not able to replace the need for this function? Can we improve it in some way?

Reply:
see my other comment.